src/cli.rs

```diff
@@ -9,6 +9,7 @@ use std::{
     path::Path,
     rc::Rc,
 };
+use ansi_term::Colour::Green;
```

Comment:
In rust its common to let [`rustfmt`](https://github.com/rust-lang/rustfmt) format the files before checking them in (usually editors handle this automatically). Rustfmt orders `use` statements alphabetically (meaning this line would be line `1`).